include/rviz_visual_tools/rviz_visual_tools.h

```diff
@@ -401,7 +401,8 @@ a   *        Warning: when using this in a loop be sure to call trigger() at end
   bool publishCone(const geometry_msgs::Pose& pose, double angle, colors color = TRANSLUCENT, double scale = 1.0);
 
   /**
-   * \brief Display a plane (a very wide, thin cuboid)
+   * \brief Display a plane. Vector (A, B, C) gives the normal to the plane.
+   *        D/|(A,B,C)| gives the distance to plane along that unit normal.
```

Comment:
```suggestion
   *    |D|/|(A,B,C)| gives the distance to plane along that unit normal.
```
Also add that you use Ax+By+Cz+D=0